### ISSUE TITLE
Lock app / note / label

### ISSUE THREAD

**Issue by qupufu** [maelchiotti/LocalMaterialNotes#243]
it would be nice to have the ability to lock notes and make them only accessible with a pin/fingerprint/etc. this could probably either be done through locking individual notes or setting a tag/label as private when those get added.

**Comment by maelchiotti** [comment]
Yes, I think I'll work on this after I've added the labels. This way I'll add full app lock / single note lock / label lock, all 3 at the same time.

**Comment by maelchiotti** [comment]
- [ ] Lock app
- [ ] Lock notes
- [ ] Lock labels
- [ ] Import locked state from DNotes exports

**Comment by maelchiotti** [comment]
- [ ] Fix back button when closing the 'Add label' dialog when the lock screen is enabled
